Source/Google Places API/GooglePlaces.swift

```diff
@@ -102,6 +115,8 @@ public class GooglePlaces: GoogleMapsService {
             completion?(response: response, error: error)
```

Comment:
I think you should set `pendingRequest = nil` before calling `completion?(response: response, error: error)`? Therefore no request is retained if it succeed